### ISSUE TITLE
Use?

### ISSUE THREAD

**Issue by richardreeve** [ScottishCovidResponse/data_pipeline_api_cpp#1]
@kzscisoft - this is the old c++ api code. Is there any use for this now, or should we archive it?

**Comment by kzscisoft** [comment]
I have not used this code at all so I am not sure if it is for me to decide this.

**Comment by alysbrett** [comment]
I think @ianhinder or Peter Fox should be able to confirm, but my understanding is that this contains only the work on a native API Qingfeng did under RAMP and it was not incorporated into either of the C++ models in the end - EERAmodel used Ian's python wrapping version and both will presumably use Kristian's native C++ version once it's ready. I think that unless Kristian is making use of it then SCRC is unlikely to use it for anything so it could be archived (and/or perhaps should be renamed to indicate that it isn't the current/active C++ API)

**Comment by ianhinder** [comment]
Yes, as far as I know, this was not used for anything past the initial exploratory phase.
